CODEOFCONDUCT.md

```diff
@@ -33,12 +33,20 @@ Harassment includes:
 * Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect vulnerable people from intentional abuse.
 * Publication of non-harassing private communication. 
 
-Ruby New Zealand prioritises marginalised people’s safety over privileged people’s comfort. Ruby New Zealand committee will not act on complaints regarding:
+The Ruby New Zealand committee reserves the right not to act on complaints about someone defending themselves from discrimination or harassment, including:
 
-* ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’
-* Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
-* Communicating in a ‘tone’ you don’t find congenial
-* Criticising racist, sexist, cissexist, or otherwise oppressive behavior or assumptions 
+* Reasonable communication of boundaries, such as "leave me alone", "go away", or "I’m not discussing this with you"
+* Criticising discriminatory or harassing attitudes or behaviour, even in a confrontational tone
+
+## Protected measures
+
+Ruby New Zealand will take measures to improve the diversity, inclusivity and accessibility of the community, including:
```

Comment:
"...will take measures to improve" seems like a strategy or planning or governance thing, rather than a code of conduct thing. Perhaps consider something along the lines of, "In all conduct of its members, Ruby New Zealand members place priority on improving the diversity, inclusivity and accessibility of the wider community."